skimage/filters/tests/test_lpi_filter.py

```diff
@@ -1,12 +1,20 @@
 import numpy as np
+import scipy
 from numpy.testing import assert_, assert_equal
 from skimage._shared import testing
+from distutils.version import LooseVersion as Version
 import unittest
 
 from skimage import data
 from skimage.filters import LPIFilter2D, inverse, wiener
+from skimage._shared._warnings import expected_warnings
 
 
+if Version(np.__version__) >= '1.15' and Version(scipy.__version__) <= '1.1.0':
```

Comment:
Are there any plans on removing this check in future? If yes, please, add a note to `TODO` file